internal/wrappers/features/index.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Index controller
+
+  Scenario: Create should create an index
+    Given Kuzzle Server is running
```

Comment:
Question: shouldn't this "Given" test be a generic hook applied to all scenarios instead?

Reply:
No because if we want to test the case where kuzzle is not running it would not be possible.
BUT I need to do something to share the kuzzle instance across all scenario. I will do this in another PR because I have to use a dependency called picocontainer and I have to do it everywhere